homeassistant/components/media_player/plex.py

```diff
@@ -62,6 +107,37 @@ def config_from_file(filename, config=None):
 
 def setup_platform(hass, config, add_devices_callback, discovery_info=None):
     """Setup the Plex platform."""
+    # optional parameters
+    optional_config = {}
+    optional_config["entity_namespace"] = config.get("entity_namespace")
```

Comment:
Correct - I did that because there is a point in the component where the config dictionary is overwritten with data pulled from plex.conf file.  I'm adding code to just merge the original config dictionary with the one pulled from plex.conf.  Don't know if thats the right way to do it but it works.